addons/friendly_tracker/XEH_preInit.sqf

```diff
@@ -0,0 +1,7 @@
+#include "script_component.hpp"
+ADDON = false;
+#include "XEH_PREP.hpp"
+
+#include "initSettings.sqf"
+
```

Comment:
Module globals should be initialized with default values here:
```sqf
GVAR(running) = false;
GVAR(markers) = [];
```

Reply:
We need to keep an eye on such things when doing transfers from mission scripts. It makes code much simpler without all these `!isNil` checks.